src/caselawclient/models/documents/__init__.py

```diff
@@ -310,6 +310,58 @@ def is_publishable(self) -> bool:
         # An empty list (which is falsy) therefore means the judgment can be published safely.
         return not self.validation_failure_messages
 
+    def check_has_only_one_version(self) -> SuccessFailureMessageTuple:
+        """Make sure the document has exactly one version."""
+        if len(self.versions) > 1:
+            return SuccessFailureMessageTuple(
```

Comment:
It's probably way too late but I keep thinking that this thing should quack like a boolean plus have additional info if false.